bedrock/mozorg/redirects.py

```diff
@@ -176,8 +176,12 @@ def decider(request, **kwargs):
     redirect(r'^m/privacy.html$', 'privacy.notices.firefox'),
 
     # Bug 1109318 /privacy/you -> privacy/tips/
+    # Bug 1238687 /privacy/tips -> teach/smarton/
     redirect(r'^privacy/you/?$',
-             'privacy.privacy-day'),
+             'teach.smarton.index'),
+    redirect(r'^privacy/tips/?$',
```

Comment:
Should we add a [test for this redirect](https://github.com/mozilla/bedrock/blob/master/test_redirects/README.md)?